packages/admin-next/dashboard/src/v2-routes/sales-channels/sales-channel-edit/sales-channel-edit.tsx

```diff
@@ -0,0 +1,31 @@
+import { Heading } from "@medusajs/ui"
+import { useAdminSalesChannel } from "medusa-react"
+import { useTranslation } from "react-i18next"
+import { useParams } from "react-router-dom"
+
+import { RouteDrawer } from "../../../components/route-modal"
+import { EditSalesChannelForm } from "../../../modules/sales-channels/sales-channel-edit/components/edit-sales-channel-form"
+
+export const SalesChannelEdit = () => {
+  const { id } = useParams()
+  const { t } = useTranslation()
+
+  const { sales_channel, isLoading, isError, error } = useAdminSalesChannel(id!)
+
+  if (isError) {
+    throw error
+  }
+
+  return (
+    <RouteDrawer>
+      <RouteDrawer.Header>
+        <Heading className="capitalize">
+          {t("salesChannels.editSalesChannel")}
+        </Heading>
+      </RouteDrawer.Header>
+      {!isLoading && sales_channel && (
```

Comment:
NIT: It's generally good practice to double-negate non-boolean fields (so `!!sales_channel` in JSX conditionals as sometimes you end up rendering some string instead of the content you intended